examples/other.md

```diff
@@ -114,84 +120,117 @@ Please note that CNCF prefers the horizontal form of our logo to the stacked ver
         <td><img src="/other/kcsp/white/kubernetes-kcsp-white.png" width="95"></td>
         <td><img src="/other/ktp/white/kubernetes-ktp-white.png" width="95"></td>
         <td><img src="/other/kss/white/kubernetes-security-specialist-white.png" width="95"></td>
-        <td><img src="/other/kcna/white/kcna_white.png" width="85"></td>
+        <td><img src="/other/kcna/white/kubernetes-kcna-white.png" width="85"></td>
+        <td>Soon</td>
         <td><img src="/other/cka/white/kubernetes-cka-white.svg" width="95"></td>
         <td><img src="/other/ckad/white/kubernetes-ckad-white.svg" width="95"></td>
         <td><img src="/other/kcsp/white/kubernetes-kcsp-white.svg" width="95"></td>
         <td><img src="/other/ktp/white/kubernetes-ktp-white.svg" width="95"></td>
         <td><img src="/other/kss/white/kubernetes-security-specialist-white.svg" width="95"></td>
-        <td><img src="/other/kcna/white/kcna_white.svg" width="85"></td>
+        <td><img src="/other/kcna/white/kubernetes-kcna-white.svg" width="85"></td>
+        <td>Soon</td>
     </tr>
 </table>
 
-#### Prometheus Certification Logos
+
+#### Cloud Native Certification Logos
 <table>
+    <tr><th colspan="9"></th></tr>
     <tr>
-        <th colspan="2">PNG</th>
-        <th colspan="1">SVG</th>
+        <th></th>
+        <th colspan="4">PNG</th>
+        <th colspan="4">SVG</th>
     </tr>
+    <tr>
+        <th></th>
+        <th>PCA</th>
+        <th>CAPA</th>
+        <th>CCA</th>
+        <th>ICA</th>
+        <th>PCA</th>
+        <th>CAPA</th>
+        <th>CCA</th>
+        <th>ICA</th>
     <tr>
         <th>color</th>
         <td><img src="/other/prometheus-certified-associate/color/PrometheusCertifiedAssociate-color.png" width="125"></td>
+        <td><img src="/other/capa/color/capa-icon-color.png" width="125"></td>
+        <td><img src="/other/cca/color/cca-icon-color.png" width="125"></td>
+        <td><img src="/other/ica/color/ica-icon-color.png" width="125"></td>
         <td><img src="/other/prometheus-certified-associate/color/PrometheusCertifiedAssociate-color.svg" width="125"></td>
+        <td><img src="/other/capa/color/capa-icon-color.svg" width="125"></td>
+        <td><img src="/other/cca/color/cca-icon-color.svg" width="125"></td>
+        <td><img src="/other/ica/color/ica-icon-color.svg" width="125"></td>
     </tr>
     <tr>
         <th>black</th>
-      <td><img src="/other/prometheus-certified-associate/black/PrometheusCertifiedAssociate-black.png" width="125"></td>
         <td><img src="/other/prometheus-certified-associate/black/PrometheusCertifiedAssociate-black.png" width="125"></td>
+        <td>Soon</td>
+        <td>Soon</td>
+        <td>Soon</td>
+        <td><img src="/other/prometheus-certified-associate/black/PrometheusCertifiedAssociate-black.png" width="125"></td>
```

Comment:
This should be an SVG

Reply:
Updated for you. I think it would have been quicker for you to just change it.